test/EFCore.SqlServer.FunctionalTests/TestUtilities/TestEnvironment.cs

```
@@ -48,26 +48,32 @@ public static bool IsFullTestSearchSupported
                     return _fullTextInstalled.Value;
                 }
 
-                using (var sqlConnection = new SqlConnection(SqlServerTestStore.CreateConnectionString("master")))
+                try
                 {
-                    sqlConnection.Open();
-
-                    using var command = new SqlCommand(
-                        "SELECT FULLTEXTSERVICEPROPERTY('IsFullTextInstalled')", sqlConnection);
-                    var result = (int)command.ExecuteScalar();
+                    using (var sqlConnection = new SqlConnection(SqlServerTestStore.CreateConnectionString("master")))
+                    {
+                        sqlConnection.Open();
 
-                    _fullTextInstalled = result == 1;
+                        using var command = new SqlCommand(
+                            "SELECT FULLTEXTSERVICEPROPERTY('IsFullTextInstalled')", sqlConnection);
+                        var result = (int)command.ExecuteScalar();
 
-                    if (_fullTextInstalled.Value)
-                    {
-                        var flag = GetFlag("SupportsFullTextSearch");
+                        _fullTextInstalled = result == 1;
 
-                        if (flag.HasValue)
+                        if (_fullTextInstalled.Value)
                         {
-                            return flag.Value;
+                            var flag = GetFlag("SupportsFullTextSearch");
+
+                            if (flag.HasValue)
+                            {
+                                return flag.Value;
+                            }
                         }
                     }
                 }
+                catch (PlatformNotSupportedException)
```

Looks like some tests should be skipped on some platforms. Hiding such errors looks perilous.

---

This condition setups environment for skipping tests. We need to try connect to server to see if we can run the test or skip.